core/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/InternalTerms.java

```diff
@@ -112,6 +113,10 @@ public long getDocCountError() {
             return docCountError;
         }
 
+        public long getBucketOrd() {
```

Comment:
Its used in the special case code just for the `terms` agg in `InternalOrder.Aggregation#comparator`.
I can move that code to some static method in this `terms` package, so it can access `InternalTerms.Bucket#bucketOrd` like it did before.
Would `TermsAggregator#comparator` be a good place for it?